src/com/group6/users/LoginView.java

```diff
@@ -0,0 +1,75 @@
+import java.util.Scanner;
+import com.group6.views.*;
+import com.group6.utils.ValidateNRIC;
+
+/**
+ * Represents the login view where users can enter their NRIC and password to authenticate.
+ * This view handles user authentication and redirects to the appropriate home view
+ * based on the user's role.
+ */
+public class LoginView implements View {

```

Comment:
Move this to `src/com/group6/views` i.e. `com.group6.views` package for consistency.